packages/js/product-editor/src/blocks/generic/toggle/edit.tsx

```
@@ -116,7 +116,9 @@
 			{ disabled && (
 				<p
 					className="wp-block-woocommerce-product-toggle__disable-copy"
-					dangerouslySetInnerHTML={ sanitizeHTML( disabledCopy ) }
+					dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)